schema/go.mod

```diff
@@ -0,0 +1,15 @@
+module github.com/opencontainers/image-spec/schema
+
+// The minimum Go release is only incremented when required by a feature.
+// At least 3 Go releases will be supported by the spec.
+// For example, updating this version to 1.19 first requires Go 1.21 to be released.
```

Comment:
This comment shouldn't be needed here -- the point of splitting this module is that we can be more aggressive about upgrading the required Go version on the `schema` module. :+1: